### PR TITLE
Minor recipe and formula fixes

### LINKED ISSUE #622: [BUG] Incorrect LCR Dimethylbenzene reaction
The LCR reaction of Benzene and Methane to form Dimethylbenzene is incorrect, it should be Benzene + 2 Methane -> Dimethylbenzene + 4 Hydrogen.
Alternatively, we could potentially do the methylation properly with something like [dimethyl sulfate](https://en.wikipedia.org/wiki/Dimethyl_sulfate)
image

Also, there was a short discussion regarding positional isomers of Xylene (dimethylbenzene) and its role in Diphenyl Isophthalate, starting here https://discord.com/channels/701354865217110096/701354865657643070/940288471669547008. Gonna put it here in case anyone wants to deal with it

### LINKED ISSUE #623: [BUG] Chlorine dupe in part of PBI Process
The offending part is the following chain of reactions:
1 Benzene + 2 Chlorine -> 1 Chlorobenzene + 1 HCl
1 PhCl + 2 Nitration Mixture -> 1 2-Nitrochlorobenzene + 1 Dil. H2SO4
1 2-NCB (+ 1/9 Cu) -> 1 3,3-Dichlorobenzidine
1 3,3-DCB + 2 Ammonia -> 1 3,3-Diaminobenzene + 2 HCl
therefore, giving 3 Cl when we started with 2.

### PR DESCRIPTION
**Pull request by m2r1k5**
**What:**
Closes #622, closes #623. 
Disables Nitrobenzene decomposition to prevent dupes
Fix Butene formula being wrong

### CODE REVIEW

**Comment by Syrcan** [comment]
In general, we have our LCR recipes void some of the secondary outputs in recipes (typically Oxygen and Hydrogen) for the sake of conveniency and gameplay balance. These recipe fixes don't need to output either of those fluids. Otherwise it sounds like a good fix.

**Comment by pablocnvs** [comment]
You could be a little more evil by quinduplicating the amount of H and remain consistent, but otherwise looks fine

**Comment by m2r1k5** [comment]
> In general, we have our LCR recipes void some of the secondary outputs in recipes (typically Oxygen and Hydrogen) for the sake of conveniency and gameplay balance. These recipe fixes don't need to output either of those fluids. Otherwise it sounds like a good fix.

To clarify, would you like me to get rid of the O and H secondary outputs?
Also, with voiding covers on the way, the convenience part won’t be as significant

File: src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ public static void register() {
         AllylChloride = new Material.Builder(1025, "allyl_chloride")
                 .fluid()
                 .color(0x87DEAA)
-                .components(Carbon, 2, Methane, 1, HydrochloricAcid, 1)
+                .components(Carbon, 3, Hydrogen, 5, Chlorine, 1)
```
Comment (TechLord22) [inline src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java:187]: The components of this compound should be kept, as we use it specifically for autogenerated decomposition. The tooltip however should be changed to `C3H5Cl`.

**Review by TechLord22 — CHANGES_REQUESTED** [review]

**Comment by ALongStringOfNumbers** [comment]
Something went wrong with the commits in this PR

**Review by TechLord22 — APPROVED** [review]

**Pull request merged**